dnd-troops-web/src/main/resources/Texts.properties

```diff
@@ -51,6 +51,21 @@ hero.search.any_role=Any role
 hero.search.any_race=Any race
 hero.search.role=Role
 
+#role
+role.all.title=Roles
+role.all.add=Add Role
+role.name=Name
+role.description=Description
+role.delete.problem=Problem while deleting role
```

Comment:
Pleas add text that only roles without assigned heroes can be deleted. It would be useful for testers